src/main/java/com/google/step/servlets/UpdateVote.java

```diff
@@ -0,0 +1,21 @@
+package com.google.step.servlets;
+
+import com.google.step.datamanager.DealVoteCountManager;
+import com.google.step.datamanager.VoteManager;
+
+public class UpdateVote {
+
+  public static void updateVote(
```

Comment:
Add a test for this?

Reply:
It is kind of embedded in the VoteServletTest, like the tests verify that the managers are called, so is there a need to have a separate test for this?

Reply:
Yeah, it's usually a good idea to add tests "closest" to the class with the behavior (even if the tests elsewhere also cover it since I could change those classes / tests and this file would no longer be tests).

Reply:
I see, okay, I added the tests already